modules/culling/src/lib/oriented-bounding-box.d.ts

```diff
@@ -17,6 +17,9 @@ export default class OrientedBoundingBox {
   // An OrientedBoundingBox of some object is a closed and convex cuboid. It can provide a tighter bounding volume than {@link BoundingSphere} or {@link AxisAlignedBoundingBox} in many cases.
   constructor(center?: readonly number[], halfAxes?: readonly number[]);
 
+  // Create OrientedBoundingBox from I3S OBB (https://github.com/Esri/i3s-spec/blob/master/docs/1.7/obb.cmn.md)
+  static fromQuaternionObb(quaternionObb: {center: number[]; halfSize: number[], quaternion: number[]}): OrientedBoundingBox;
```

Comment:
If you compare to other classes in math.gl, I think you will see our "overloaded constructors" are not static but called on the instance. 

https://github.com/uber-web/math.gl/blob/master/modules/culling/src/lib/bounding-sphere.d.ts#L14

Let's follow the existing design choice.

For the name, another option would be `fromCenterHalfSizeQuaternion()`